README.md

```diff
@@ -48,23 +48,25 @@ Variants that require different screen sizes and window addressing offsets are n
 
 ## Troubleshooting
 
-Refer to the [troubleshooting guide](https://github.com/almindor/mipidsi/blob/master/docs/TROUBLESHOOTING.md) if you experience problems like a blank screen or incorrect colors.
+Refer to the [troubleshooting guide](./docs/TROUBLESHOOTING.md) if you experience problems like a blank screen or incorrect colors.
```

Comment:
This link isn't valid anymore, because the troubleshooting guide has just been moved into the docs (#176). The new link should be https://docs.rs/mipidsi/latest/mipidsi/_troubleshooting/index.html, once the next version of the crate is released.